src/types/index.ts

```diff
@@ -298,3 +303,38 @@ export interface PreInvoice {
   channelId: string;
   nextLocalBalance: number;
 }
+
+export interface Activity {
+  id: number;
+  source: LightningNode;
+  destination: LightningNode;
+  intervalSecs: number;
+  amountMsat: number;
+}
+
+export interface Simulation {
+  networkId: number;
```

Comment:
Do we need to store the `networkId` here? The `Simulation` object is nested inside the `Network` which already has an `id` that can be referenced.